ResizableMKCircleOverlay/src/main/java/org/moe/ui/CustomMKCircleOverlay.java

```diff
@@ -175,9 +171,12 @@ public void drawMapRectZoomScaleInContext(@ByValue MKMapRect mapRect, @NFloat do
                 radiusAtLatitude, 0, 2 * Math.PI, 0);
         CoreGraphics.CGContextDrawPath(ctx, CGPathDrawingMode.FillStroke);
 
-        if(delegate != null) {
-            Globals.dispatch_async(Globals.dispatch_get_main_queue(), () -> {
-                delegate.onRadiusChange(mapRadius);
+        if (delegate != null) {
+            Globals.dispatch_async(Globals.dispatch_get_main_queue(), new Globals.Block_dispatch_async() {
```

Comment:
This kind of workaround is no longer needed, lambda should be supported by the latest 2.0-SNAPSHOT version.

See multi-os-engine/multi-os-engine#146

Reply:
> This kind of workaround is no longer needed, lambda callbacks should be supported by the latest 2.0-SNAPSHOT version.

Is it already published to sonatype?

Reply:
> > This kind of workaround is no longer needed, lambda callbacks should be supported by the latest 2.0-SNAPSHOT version.
> 
> Is it already published to sonatype?

Yep

Reply:
That's odd, it works fine on my machine. Are you sure you are using the latest snapshot plugin & sdk from sonatype instead of maven local or a local dev sdk?

Reply:
Ahh yes, I cleared the gradle cache but I forgot to clear the local maven repo... I currently can't test it because my IPhone is not working, but it will probably work now.